notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>geoplot isn't part of the pudl-dev environment, and doesn't seem to be getting used below. Do we need it? It caused this cell to fail for me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Oh yeah, that was leftover from some other maps I that didn't make the cut</p>